### ISSUE TITLE
Non-Predictive Option?

### ISSUE THREAD

**Issue by jamiefranks** [danielshorten/k9t9#28]
Not an issue per say but not sure how else to contact you, so apologies about this.

Is there an option for non-predictive text, where you press the Key x amount of times for each letter? For example, Press Key 2 three times for the letter C?

**Comment by duomits** [comment]
Long press the pound key to change entry modes. 

It cycles through Predictive > Letter Entry > Number Entry

Check the [README.md](https://github.com/danielshorten/k9t9/blob/main/README.md)

**Comment by jamiefranks** [comment]
My apologies, must've missed that! Is there a way to set it to Letter Entry for default? As it always resets to predictive?

**Comment by danielshorten** [comment]
[Version 0.12.0](https://github.com/danielshorten/k9t9/releases/tag/0.12.0) has a feature to preserve the most recent text entry mode.

**Comment by joananeves** [comment]
I'm sorry to use this to make a question, is there any way to get this app though Google Play?
My duoqin f22 doesn't allow adb install

**Comment by cjmabry** [comment]
@joananeves I don't believe so, but you should be able to visit the releases page in Github using the web browser on your phone and manually download and install the APK here: https://github.com/danielshorten/k9t9/releases

As of now, the latest packaged release is v0.12.0, and you can find it under Assets on the releases page.

**Comment by joananeves** [comment]
Thanks @cjmabry !
I tried that, but my device doesn't allow apk to be installed with adb.
And I don't want to try to root the mobile...
